### ISSUE TITLE
performance on TIMIT? optimized parameters?

### ISSUE THREAD

**Issue by fanskyer** [jonrein/tensorflow_CTC_example#5]
tried to run a TIMIT experiment, 39 phonemes. Used all the original parameters. after 120 epochs, loss is at 120, and error rate at 0.9. So can you give me some guidance on how to obtain a decent performance using this repo? I didn't do feature mean/variance normalization. is this a must?

thanks

**Comment by jonrein** [comment]
I haven't tried running without normalization, so I'm not sure about "must", but it is strongly recommended.  Let me know if that doesn't help things.  Training on a 39-phoneme target encoding may take more time than the 27-phoneme encoding I've run with, so you may want to let it run a little longer.

**Comment by fanskyer** [comment]
Thanks! I noticed you did the normalization in that toy example, after the feature normalization, now the model seems converge. have you tried tune the parameters, model complexity to obtain some better results? thanks

**Comment by jonrein** [comment]
No, I didn't make any real efforts at optimization.  The primary goal for me was to replicate the Graves 2012 experiment to benchmark the TensorFlow implementation against a relatively straightforward published result with LSTM/CTC.

**Comment by fanskyer** [comment]
got it, thanks
